include/VCR_Globals.h

```diff
@@ -17,8 +17,19 @@ extern VCRData_s vcr_data; // NOLINT
 
 /* ADC setup */
 constexpr unsigned int channels_within_mcp_adc = 8;
+struct ADCInstance
```

Comment:
nit: this naming implies there's one ADC. not sure what naming would be better, but please rename to be descriptive

Reply:
Actually I change my mind, I really don't like this name. This isn't an instance-- this isn't even a singleton type. Name it ADCBundle or ADCsOnVCR or something more descriptive.